albam/engines/mtfw/mesh.py

```diff
@@ -53,8 +55,180 @@
     6: Mod156.Vertex5,
     7: Mod156.Vertex5,
     8: Mod156.Vertex5,
+    # 0x4325a03e: Mod21.Vertex4325,  # IANonSkinTBN_4M shape keys not implemented yet
+    # 0x2f55c03d: Mod21.Vertex2f55,  # IASkinOTB_4WT_4M shape keys not implemented yet
+    0xa14e003c: Mod21.VertexA14e,  # IANonSkinBCA
+    0x2082f03b: Mod21.Vertex2082,  # IANonSkinBLA
+    0xc66fa03a: Mod21.VertexC66f,  # IANonSkinBA
+    0xd1a47038: Mod21.VertexD1a4,  # IANonSkinBL
+    0x207d6037: Mod21.Vertex207d,  # IANonSkinBC
+    0xa7d7d036: Mod21.VertexA7d7,  # IANonSkinB
+    0x37a4e035: Mod21.Vertex37a4,  # IANonSkinTBNLA
+    0xb6681034: Mod21.VertexB668,  # IANonSkinTBNCA
+    0x9399c033: Mod21.Vertex9399,  # IANonSkinTBCA
+    0x12553032: Mod21.Vertex1255,  # IANonSkinTBLA
+    0x747d1031: Mod21.Vertex747d,  # IANonSkinTBNA
+    0x63b6c02f: Mod21.Vertex63b6,  # IANonSkinTBNL vertex alpha
+    0x926fd02e: Mod21.Vertex926f,  # IANonSkinTBNC
+    0xafa6302d: Mod21.VertexAfa6,  # IANonSkinTBA
+    0x5e7f202c: Mod21.Vertex5e7f,  # IANonSkinTBN
+    0xb86de02a: Mod21.VertexB86d,  # IANonSkinTBL vertex alpha
+    0x49b4f029: Mod21.Vertex49b4,  # IANonSkinTBC
+    0xd8297028: Mod21.Vertex8297,  # IANonSkinTB
+    0xcbcf7027: Mod21.VertexCbcf,  # IASkinTBNLA8wt
+    0xd84e3026: Mod21.VertexD84e,  # IASkinTBC8wt
+    0x75c3e025: Mod21.Vertex75c3,  # IASkinTBN8wt
+    0xbb424024: Mod21.VertexBb42,  # IASkinTB8wt
+    0x64593023: Mod21.Vertex6459,  # IASkinTBNLA4wt
+    0x77d87022: Mod21.Vertex77d8,  # IASkinTBC4wt
+    0xdA55a021: Mod21.VertexDa55,  # IASkinTBN4wt
+    0x14d40020: Mod21.Vertex14d4,  # IASkinTB4wt
+    0xb392101f: Mod21.VertexB392,  # IASkinTBNLA2wt
+    0xa013501e: Mod21.VertexA013,  # IASkinTBC2wt
+    0xd9e801d: Mod21.VertexD9e8,  # IASkinTBN2wt
+    0xc31f201c: Mod21.VertexC31f,  # IASkinTB2wt
+    0xd877801b: Mod21.VertexD877,  # IASkinTBNLA1wt
+    0xcbf6c01a: Mod21.VertexCbf6,  # IASkinTBC1wt
+    0x667b1019: Mod21.Vertex667b,  # IASkinTBN1wt
+    0xa8fab018: Mod21.VertexA8fa,  # IASkinTB1wt
+    0xa320c016: Mod21.VertexA320,  # IASkinBridge8wt
+    0xcb68015: Mod21.VertexCb68,  # IASkinBridge4wt
+    0xdb7da014: Mod21.VertexDb7d,  # IASkinBridge2wt
+    0xb0983013: Mod21.VertexB098,  # IASkinBridge1wt
 }
 
+VERTEX_FORMATS_RGBA = (
```

Comment:
For most of these groups It might have been easier to check if the vertex format has the attribute itself instead of classifying them before-hand.
E.g., `if hasattr(vertex_format, "uv3")`,  `if hasattr(vertex_format, "tangent¨)`  directly.

Maybe something to refine later.